lib/lookup/Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 publish = false
 
 [dependencies]
-serde = { version = "1.0.136", default-features = false, features = ["derive"] }
+serde = { version = "1.0.136", default-features = false, features = ["derive", "alloc"] }
```

Comment:
There is a `visit_string` deserialize method that is implemented in this crate, but it requires the `alloc` feature to be enabled because it (as implied) allocates a `String`.

The reason this doesn't trigger the CI is that almost all other crates that depend on this one have serde's `std` feature enabled, which implies `alloc`, and thus it gets enabled for the downstream crate as well.

However, when working with the `value` crate, it has no dependencies other than `lookup`, and thus when building that crate, it would result in a compilation error. This fixes that.